role_manager_b_test.go

```diff
@@ -0,0 +1,94 @@
+package casbin
+
+import (
+	"fmt"
+	"testing"
+
+)
+
+func BenchmarkRoleManagerSmall(b *testing.B) {
+	e, _ := NewEnforcer("examples/rbac_model.conf", false)
+	// Do not rebuild the role inheritance relations for every AddGroupingPolicy() call.
+	e.EnableAutoBuildRoleLinks(false)
+
+	// 100 roles, 10 resources.
+	pPolicies := make([][]string, 0)
+	for i := 0; i < 100; i++ {
+		pPolicies = append(pPolicies, []string{fmt.Sprintf("group%d", i), fmt.Sprintf("data%d", i/10), "read"})
+	}
+	e.AddPolicies(pPolicies)
+
+	// 1000 users.
+	gPolicies := make([][]string, 0)
+	for i := 0; i < 1000; i++ {
+		gPolicies = append(gPolicies, []string{fmt.Sprintf("user%d", i), fmt.Sprintf("group%d", i/10)})
+	}
+
+	e.BuildRoleLinks()
+
+	rm := e.GetRoleManager()
+
```

Comment:
Should here be b.ResetTimer() ？we just need the time for HasLink

Reply:
Yes, please add `b.ResetTimer()` here.